youtube_dl/extractor/nuevo.py

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+from ..utils import float_or_none
+
+class NuevoBaseIE(InfoExtractor):
+    def _extract_nuevo(self, config_url, video_id, info=None, ignore_hd=False):
```

Comment:
@dstftw: I added the ignore_hd parameter because trutube's filehd always 404s (and that was mentioned in the extractor's code). The original extractor also ignored it for this reason. I removed the parameter for now and made the trutube extractor remove the bogus hd format from the list instead, but that's a little ugly. The test fails without this.